src/CLI/clitree/cli-xml/vrf.xml

```diff
@@ -25,12 +25,12 @@ limitations under the License.
     <VIEW name="enable-view">
       <COMMAND name="show ip vrf" help="Show vrf information">
         <ACTION> 
-          python $SONIC_CLI_ROOT/openconfig_network_instance_api.py get_list_openconfig_network_instance_network_instances_network_instance show_mgmt_vrf.j2
+          python $SONIC_CLI_ROOT/sonic-cli-vrf.py get_openconfig_network_instance_network_instances_network_instance show_vrf.j2
         </ACTION>
       </COMMAND>
       <COMMAND name="show ip vrf management" help="Show management vrf information">
```

Comment:
Capitalize VRF in the `help` string here as well, and across the entire XML file. It's important to have proper formatting and be consistent throughout. I know this doesn't fall under your diffs, but thanks for taking care of it anyway.

Reply:
Done